### DISCUSSION TITLE
Build Failed while cloning and building MirthConnect from github repository

### DISCUSSION THREAD

**Discussion by sal4455** [nextgenhealthcare/connect#5242] [Q&A]
Hi All,
I tried to clone this github repository https://github.com/nextgenhealthcare/connect and build again after doing some changes in build.xml files. 
I referred this to clone and build from source (https://github.com/nextgenhealthcare/connect/wiki/Clone-Build-and-Run-from-Source)
But on building again, I'm getting below error.
[mirth_build_error.txt](https://github.com/nextgenhealthcare/connect/files/8891798/mirth_build_error.txt)

**Comment by tonygermano** [comment]
What JDK are you using to build? I believe the instructions were written to build with Oracle Java 8.

**Reply by sal4455** [reply]
yes, I using JDK1.8 and followed the instructions to build. 
Also I referred this - https://github.com/nextgenhealthcare/connect/discussions/5156, now I'm getting below error 
<img width="621" alt="image" src="https://user-images.githubusercontent.com/104156028/173878429-f2156ee4-d7e2-4f52-88c7-0b23d52d1f8b.png">

Inside server/build.xml - 
<img width="926" alt="image" src="https://user-images.githubusercontent.com/104156028/173877715-e42833af-16f4-4a9f-a445-9d3566f50d92.png">

**Reply by ab-20-m** [reply]
The testclasspath is longer than the maximum command line length limit on Windows. You will have to modify the build script to create a classpathing JAR file and use that for the testclasspath. [Example blog post](https://vvratha.blogspot.com/2012/04/avoiding-long-classpath-issue-in-ant.html).

**Reply by JoeFox82** [reply]
Anyone here that can share the edited build.xml? I've the same problem and no experience using ant  scripting :-(
